jinxify/src/app/page.tsx

```diff
@@ -1,30 +1,87 @@
-import { SignIn } from "@/components/auth/sign-in";
-import { Button } from "@/components/ui/button";
-import {
-	Card,
-	CardContent,
-	CardDescription,
-	CardFooter,
-	CardHeader,
-	CardTitle,
-} from "@/components/ui/card";
-import Image from "next/image";
+import { Metadata } from "next"
+import Image from "next/image"
+import Link from "next/link"
+
+import { UserAuthForm } from "../components/auth/user-auth-form"
+import Icon from "../components/icons/logo-icon";
+import { cn } from "@/lib/utils";
+import { buttonVariants } from "@/components/ui/button";
+
+export const metadata: Metadata = {
+  title: "Authentication",
+  description: "Authentication forms built using the components.",
+}
 
 export default function Home() {
-	return (
-		<Card>
-			<CardHeader>
-				<CardTitle>jinxify</CardTitle>
-				<CardDescription>Card Description</CardDescription>
-			</CardHeader>
-			<CardContent>
-				<p>Card Content</p>
-				<Button>Click Me!</Button>
-				<SignIn />
-			</CardContent>
-			<CardFooter>
-				<p>Card Footer</p>
-			</CardFooter>
-		</Card>
-	);
+  return (
+    <>
+      <div className="container relative h-screen flex-col items-center justify-center p-4 md:p-0 grid max-w-full lg:max-w-none lg:grid-cols-2 lg:px-0">
+        <div className="absolute left-6 top-8">
+			<Icon className="h-12 w-12 mr-2" />
+		</div>
+		<Link
```

Comment:
If we are only using github, google authentication then i think it doesn't make sense to have a separate login button. We can just say in this page something like "login or signup to use our app with one of these methods"